website/data/docs-nav-data.json

```diff
@@ -730,11 +730,19 @@
             "path": "datasources/hcp"
           },
           {
-            "title": "Iteration",
+            "title": "Version",
+            "path": "datasources/hcp/hcp-packer-version"
+          },
+          {
+            "title": "Artifact",
+            "path": "datasources/hcp/hcp-packer-artifact"
+          },
+          {
+            "title": "Iteration (Deprecated)",
```

Comment:
I'm not sure how I feel about having Deprecated in the nav bar but lets see what it looks like.